canonical/jp.ubuntu.com
# fix(branding): update "Ubuntu Advantage" to "Ubuntu Pro" #6276

## Done

- updated "Ubuntu Advantage" copy to "Ubuntu Pro" (and one instance of "Ubuntu Advantage for Infrastructure" to "Ubuntu Pro (Infra-only)")

## QA

- Check out this branch
- Search the codebase for "Ubuntu Advantage", there should be no results

Fixes https://github.com/canonical/web-squad/issues/6276
